bom/pom.xml

```diff
@@ -99,6 +99,105 @@ quarkus.vertx.prefer-native-transport=true
 quarkus.vertx.resolver.cache-max-time-to-live=0
     </quarkus.application.properties>
 
+    <!-- Local dev and test mode  support -->
+    <!-- Fixed ports used when starting hono services/adapters locally or with fixed ports -->
+    <fixed.auth.amqps.port>5673</fixed.auth.amqps.port>
+    <fixed.auth.amqp.port>5674</fixed.auth.amqp.port> <!-- Reserved - not used -->
+    <fixed.deviceregistry.amqps.port>5675</fixed.deviceregistry.amqps.port> <!-- Reserved - not used -->
+    <fixed.deviceregistry.amqp.port>5676</fixed.deviceregistry.amqp.port>
+    <fixed.deviceregistry.http.port>1080</fixed.deviceregistry.http.port>
+    <fixed.deviceregistry.https.port>1443</fixed.deviceregistry.https.port> <!-- Reserved - not used -->
+    <fixed.commandrouter.amqps.port>5677</fixed.commandrouter.amqps.port> <!-- Reserved - not used -->
+    <fixed.commandrouter.amqp.port>5678</fixed.commandrouter.amqp.port>
+
+    <fixed.http.adapter.port>8080</fixed.http.adapter.port>
+    <fixed.https.adapter.port>8443</fixed.https.adapter.port>
+    <fixed.mqtt.adapter.port>1883</fixed.mqtt.adapter.port>
+    <fixed.mqtts.adapter.port>8883</fixed.mqtts.adapter.port>
+    <fixed.amqp.adapter.port>5672</fixed.amqp.adapter.port>
+    <fixed.amqps.adapter.port>5671</fixed.amqps.adapter.port>
+    <fixed.coap.adapter.port>5683</fixed.coap.adapter.port>
+    <fixed.coaps.adapter.port>5684</fixed.coaps.adapter.port>
+    <fixed.lora.adapter.port>2080</fixed.lora.adapter.port>
+    <fixed.loras.adapter.port>2443</fixed.loras.adapter.port>
+
+    <quarkus.application-dev.properties>
+hono.auth=${hono.auth}
+hono.auth.host=localhost
```

Comment:
does this mean that all components started locally expect the Auth Server to also have been started locally (and not on Docker)?

Reply:
Then you have the option to configure the host using system properties. But in default case it would work out of the box